pkg/terraform/stages/aws/stages.go

```diff
@@ -17,6 +21,18 @@ var PlatformStages = []terraform.Stage{
 		"aws",
 		"bootstrap",
 		[]providers.Provider{providers.AWS},
-		stages.WithNormalBootstrapDestroy(),
+		stages.WithCustomBootstrapDestroy(customBootstrapDestroy),
 	),
 }
+
+func customBootstrapDestroy(s stages.SplitStage, directory string, terraformDir string, varFiles []string) error {
+	opts := make([]tfexec.DestroyOption, 0, len(varFiles)+1)
+	for _, varFile := range varFiles {
+		opts = append(opts, tfexec.VarFile(varFile))
+	}
+	opts = append(opts, tfexec.Refresh(false))
```

Comment:
Can we add a comment + a debug message to say we are changing default behavior by specifically setting refresh=False because that was resulting in Terraform trying to describe fields (specifically security group rules) which is not an allowed operation in some regions like CS2. So, will help us in the future if we hit some unintended consequences of setting refresh=false.